bot.py

```diff
@@ -63,6 +65,42 @@ def get_token():
 		return "".join(f.readlines()).strip()
 
 
+@bot.command()
+async def minigame(ctx: commands.Context):
+	"""A minigame to allow users to aquire Pokemon"""
+	await lock.acquire()
+
+	await bot.change_presence(activity=discord.Game("Who's That Pokemon?"))
+
+	pokemon = await battleapi.generate_pokemon()
+	name = pokemon.Name
+	await ctx.send("Can you guess the name of the Pokemon shown below?")
+	await ctx.send(
+		file=discord.File(f"images/{pokemon.NatDex}.png", filename="whosthatpokemon.png")
+	) #TODO generate silhouette images
+	await ctx.send("Please prefix all guesses with \"guess\"")
+
+	def check(m):
+		return m.id != bot.user.id and m.channel == ctx.channel and m.content.startswith(
+			"guess"
+		)
```

Comment:
Why not just wait for the correct guess?

Reply:
responding to incorrect guesses makes the minigame more interactive in my opinion. plus i'm going to let people know if their guesses are close but not quite right